Sources/SwiftSyntax/generated/TokenKind.swift

```diff
@@ -14,7 +14,7 @@
 
 /// Enumerates the kinds of tokens in the Swift language.
 public enum TokenKind: Hashable {
-  case eof
+  case endOfFile
```

Comment:
Hello @mininny 👋
The subfiles in the directory `generated` are automatically generated through `CodeGeneration`.
You need to change the code in `CodeGeneration` and refer to the instructions in these documents to generate the code 🙂 
https://github.com/apple/swift-syntax/blob/main/CONTRIBUTING.md#generating-source-code
https://github.com/apple/swift-syntax/blob/main/CodeGeneration/README.md

Reply:
I see, thanks! I'll rebuild the project with codegen. It's my first time contributing here so I didn't realize it. Please disregard this PR for now :)